README.md

```diff
@@ -5,3 +5,14 @@ The Truck Route Planner application for Translink built using the Simple Map Kit
 
 ## Updated OCP4 Deployment Steps
 Builds and Deployment Templates are provided in the Openshift Dir located in the root of this repo
+
+
+## Testing locally
+
+Add this entry to `/etc/hosts`:
+
+    127.0.0.1       translink-d.apps.gov.bc.ca
+
+Start the app with `npm start`
```

Comment:
Thanks for adding this section. Consider suggesting a run of 'npm install' before 'npm start'.